static/js/zamboni/validator.js

```diff
@@ -343,11 +343,13 @@ function initValidator($doc) {
                     }
                 });
                 $context.append($code);
-            } else if (msg.line) {
+            } else if (msg.line && msg.column !== 'undefined') {
```

Comment:
shouldn't this just be `msg.column !== undefined`? I can be totally wrong though 😆